<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelResp">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>255</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Student name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditName">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Adam</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelID">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>80</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Student ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditID">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>4527</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHistoryMark">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>96</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>History Mark</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditGeogMark">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>95</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelGeogMark">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>112</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Geogrophy Mark</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditHistoryMark">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>90</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
